File: tournaments/primeFactors/primeFactors.cpp
std::vector<int> primeFactors(int n) {
  std::vector<int> factors;
  int divisor = 2;

  while (n >= 2) {
    if (n % divisor == 0) {
      factors.push_back(divisor);
      n /= divisor;
    }
    else {
      divisor++;
    }
  }
  return factors;
}

File: tournaments/primeFactors/primeFactors.js
function primeFactors(n) {
    const result = [];
    let a = 2;

    while (2 <= n) {
        if (n % a) ++a;
        else {
            result.push(a);
            n = ~~(n / a);
        }
    }

    return result;
}
